app/templates/src/app.js

```diff
@@ -1,18 +1,16 @@
-import Map from "can/map/";
-import route from "can/route/";
-import 'can/map/define/';
-import 'can/route/pushstate/';
+import DefineMap from 'can-define/map/';
+import route from 'can-route';
+import 'can-route-pushstate';
 
-const AppViewModel = Map.extend({
-  define: {
-    message: {
-      value: 'Hello World!',
-      serialize: false
-    },
-    title: {
-      value: '<%= name %>',
-      serialize: false
-    }
+const AppViewModel = DefineMap.extend({
+  route: "string",
```

Comment:
In addition to what @marshallswain said above, can.route has *always* added a `route` property to the application view model.  It's because of us now using can-define that we need to predefine properties (so that they are observable).

Note that this is a prerelease requirement but might go away. can-define also has a `.set(key, val)` method that is observable, so can-route might be able to use this method to set its route prop.